drmemory/drmemory.c

```diff
@@ -1508,6 +1510,7 @@ create_global_logfile(void)
         f_suppress = open_logfile("suppress.txt", false, -1);
         f_potential = open_logfile(RESULTS_POTENTIAL_FNAME, false, -1);
         print_version(f_potential, true);
+        f_fuzz = open_logfile("fuzz_results.txt", false, -1);
```

Comment:
Should this only be opened when fuzzing is active, to avoid an empty file?  Or an empty file is a feature: remind new users fuzzing is available.